packages/twenty-server/src/modules/connected-account/auto-companies-and-contacts-creation/services/create-company-and-contact.service.ts

```diff
@@ -179,18 +164,31 @@ export class CreateCompanyAndContactService {
           return;
         }
 
-        const calendarEventParticipantsWithoutPersonIdAndWorkspaceMemberId =
-          await this.calendarEventParticipantRepository.getWithoutPersonIdAndWorkspaceMemberId(
-            workspaceId,
-            transactionManager,
-          );
-
-        await this.calendarEventParticipantService.updateCalendarEventParticipantsAfterContactCreation(
-          calendarEventParticipantsWithoutPersonIdAndWorkspaceMemberId,
+        await this.calendarEventParticipantService.updateCalendarEventParticipantsAfterPeopleCreation(
           workspaceId,
           transactionManager,
         );
       },
     );
   }
+
+  async createCompaniesAndContactsAndUpdateParticipantsAfterSync(
```

Comment:
This method seems really close to createCompaniesAndContactsAndUpdateParticipants, I think we can refactor it in a better way